kornia/augmentation/random_generator/_2d/random_rain.py

```diff
@@ -74,3 +77,60 @@ def forward(self, batch_shape: tuple[int, ...], same_on_batch: bool = False) ->
             'drop_height_factor': drop_height_factor,
             'drop_width_factor': drop_width_factor,
         }
+
+    def random_rain_augmentation(
+        self,
+        image: torch.Tensor,
+        num_raindrops: int = 100,
+        min_length: int = 5,
+        max_length: int = 15,
+        color: torch.Tensor = torch.tensor([255]),
+    ) -> torch.Tensor:
+        """Apply random rain augmentation to the input image.
+        Args:
+            image (torch.Tensor): the input image with shape :math:`(C,H,W)`.
+            num_raindrops (int, optional): number of raindrops to draw. Defaults to 100.
+            min_length (int, optional): minimum length of a raindrop. Defaults to 5.
+            max_length (int, optional): maximum length of a raindrop. Defaults to 15.
+            color (torch.Tensor, optional): the color of the raindrops. Defaults to torch.tensor([255]).
```

Comment:
These will be injected automatically by a sphinx plugin
```suggestion
        Args:
            image: the input image with shape :math:`(C,H,W)`.
            num_raindrops: number of raindrops to draw.
            min_length: minimum length of a raindrop.
            max_length: maximum length of a raindrop.
            color: the color of the raindrops.
```

Reply:
+1